container/stream/streams.go

```diff
@@ -119,28 +119,31 @@ func (c *Config) CopyToPipe(iop *cio.DirectIO) {
 	ctx := context.TODO()
 
 	c.dio = iop
-	copyFunc := func(w io.Writer, r io.ReadCloser) {
+	copyFunc := func(name string, w io.Writer, r io.ReadCloser) {
 		c.wg.Add(1)
 		go func() {
 			if _, err := pools.Copy(w, r); err != nil {
-				log.G(ctx).Errorf("stream copy error: %v", err)
+				log.G(ctx).Errorf("stream copy %s error: %v", name, err)
```

Comment:
Not critical, but as we're updating this, we could consider using structured logs / fields for these, e.g.;


```go
log.G(ctx).WithFields(log.Fields{"error": err, "stream": name}).Error("copy stream")
```


☝️ open to suggestions for the message itself; not sure if "stream" is the right term to use for all of these